packages/xstate-vue/package.json

```diff
@@ -58,19 +58,17 @@
   },
   "devDependencies": {
     "@testing-library/jest-dom": "^5.11.9",
-    "@testing-library/vue": "^6.4.0",
-    "@vue/compiler-sfc": "^3.0.7",
```

Comment:
Removed this because it's a built-in export in v3 and it doesn't seem like we're using this directly anywhere. Not sure if that's something we should check in the build output (though building works fine).